humhub/protected/config/web.php

```diff
@@ -1,9 +0,0 @@
-<?php
```

Comment:
because the config directory is mapped as volume during container-creation it is likely (depends on volume-driver) that the original content of the directory is hidden and the container is initially empty. Because of that I added this default file to replicate a "normal" config-dir after the first container startup. I would recommend keeping this file.

Reply:
Will do. I wasn't sure whether I had introduced this file on my own as part of many attempts at getting `baseUrl` working with a sub-directory so I removed it.